### ISSUE TITLE
yify-subs APIs not working.

### ISSUE THREAD

**Issue by qadirsuh** [oussama1598/yify-subs#1]
api.yifysubtitles.com and http://www.yifysubtitles.com/subtitle-api/ant-man-yify-72187.zip not working.
looks APIs not working. why?

I tried to download a subtitle via imdb id i.e http://api.yifysubtitles.com/subs/0478970 but I got 404 page not availble.

I also tried to run your code i.e test.js but I got this response
https://www.dropbox.com/s/a1sqf0xy3cm7zmo/Screenshot%202017-04-12%2011.17.49.png?dl=0

Any workaround please

**Comment by oussama1598** [comment]
yeah you are right, the yify api is no longer available that's why ! the only way you can work around it is by making a scraper which will work 100% in this case.

 it's been long since i updated this repo i think now it's the time!

**Comment by qadirsuh** [comment]
Are you going make an scrapper for this?
